types/node/ts4.0/tslint.json

```diff
@@ -0,0 +1,13 @@
+// TODO: duplicate or extend?
```

Comment:
I copied this from `types/node/tslint.json`. I wasn't sure whether to duplicate or use `extends`?